### PR TITLE
testing/xmrig-proxy: new aport

### PR DESCRIPTION
**Pull request by OTLabs**
https://github.com/xmrig/xmrig-proxy
XMRig Proxy is a high performance Monero (XMR) Stratum proxy

### CODE REVIEW

File: testing/xmrig-proxy/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+check() {
+	msg 'Running tests...'
+	cd "$builddir"/build
+	./xmrig-proxy --version
+}
+
+package() {
+	msg2 'Packaging binary...'
```
Comment (andypost) [inline testing/xmrig-proxy/APKBUILD]: this  and similar lines should be removed

File: testing/xmrig-proxy/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+arch="all"
+license="GPL-3.0"
+depends="libuv libmicrohttpd openssl libuuid"
+makedepends="cmake libuv-dev libmicrohttpd-dev openssl-dev util-linux-dev"
+subpackages="$pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/xmrig/xmrig-proxy/archive/v$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
+
+prepare() {
+	default_prepare
```
Comment (andypost) [inline testing/xmrig-proxy/APKBUILD]: no reason in that

**Comment by OTLabs** [comment]
Addressed all your recommendations @andypost Thank you!

**Comment by OTLabs** [comment]
Update APKBUILD based on recommendations received for xmrig: removed dependencies and fixed install script.

libuv dependency was not traced as in xmrig. Should I add it manually?

**Comment by andypost** [comment]
Commited ed291c9d

**Pull request closed without merging**